common/service/config/config.go

```diff
@@ -110,6 +110,8 @@ type (
 		TLS RootTLS `yaml:"tls"`
 		// Metrics is the metrics subsystem configuration
 		Metrics *Metrics `yaml:"metrics"`
+		// Settings for authentication and authorization
+		Security Security `yaml:"security"`
```

Comment:
[Nit] I don't have a proposed suggestion, but security seems too generic of a name here.

Reply:
Rename to `Authorization`. We might need to add `Security` later and move `Authorization` and `TLS` under it.